File: src/main/java/cal/fstg/java/controller/ResumeController.java
package cal.fstg.java.controller;

//import cal.fstg.java.model.Document;
//import cal.fstg.java.model.Resume;
//import cal.fstg.java.service.DocumentService;
//import cal.fstg.java.service.ResumeService;
//import org.springframework.beans.factory.annotation.Autowired;
//import org.springframework.web.bind.annotation.*;
//
//import java.util.List;
//import java.util.Optional;

//@RestController
//@RequestMapping("/resumes")
//public class ResumeController {
//
//    private final ResumeService resumeService;
//
//    @Autowired
//    public ResumeController(ResumeService resumeService) {
//        this.resumeService = resumeService;
//    }
//
//    @GetMapping("/get/all")
//    public List<Resume> getAll() {
//        return resumeService.getAll();
//    }
//
//    @GetMapping("/get/{id}")
//    public Optional<Resume> getOne(@PathVariable long id) {
//        return resumeService.getOne(id);
//    }
//
//    @PostMapping("/save")
//    public void save(@RequestBody Resume resume) {
//        resumeService.save(resume);
//    }
//
//    @DeleteMapping("/delete/{id}")
//    public void delete(@PathVariable long id) {
//        resumeService.delete(id);
//    }
//}

File: src/main/kotlin/cal/fstg/stagerkotlin/controller/ResumeController.kt
package cal.fstg.stagerkotlin.controller

import cal.fstg.stagerkotlin.model.Resume
import cal.fstg.stagerkotlin.model.Student
import cal.fstg.stagerkotlin.service.DocumentService
import cal.fstg.stagerkotlin.service.ResumeService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping("/resmumes")
class ResumeController(@Autowired private val resumeService: ResumeService) {

    @GetMapping("/get/all")
    fun getAll(): List<Resume> = resumeService.getAll()

    @GetMapping("/get/{id}")
    fun getOne(@PathVariable id: Long) = resumeService.getOne(id)

    @PostMapping("/save")
    fun save(@RequestBody resume: Resume) = resumeService.save(resume)

    @DeleteMapping("/delete/{id}")
    fun delete(@PathVariable id: Long) = resumeService.delete(id)
}